lib/support/geometry/GroupD8.js

```diff
@@ -53,17 +53,17 @@ class GroupD8
 	}
 	
 	/**
-	 * isSwapWidthHeight
+	 * isVertical
 	 * This function is used in order to know if width and height needs to be swapped. It appends sometimes.

```

Comment:
Maybe you should adapt the commentary here to match with the function naming.
Swapping width and height is a consequence of the return of the function, not the purpose of the function.